src/contents/en-gb/parsec.yaml

```diff
@@ -0,0 +1,63 @@
+header:
+  title: PARSEC
+pageIntro:
+  title: What is PARSEC?
+  desc: PARSEC is a Protocol for Asynchronous, Reliable, Secure and Efficient Consensus and is the world's first open source ABFT protocol that can be used in a truly permissionless setting created by MaidSafe.

```

Comment:
Suggest link from `ABFT protocol` to what that means

Reply:
Expanded out the initialism